<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1457</width>
    <height>764</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1457</width>
    <height>764</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1457</width>
    <height>764</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BtWaehlen">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>670</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Datei wählen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtAuswerten">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>670</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Auswerten</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>630</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Keine Datei ausgewählt</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QChartView" name="graphicsiew_rechts">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>701</width>
      <height>451</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QChartView" name="graphicsview_links">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>711</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Achsen_Auswahl_Links">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Wert_Auswahl_Links">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Achsen_Auswahl_Rechts">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>470</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="Wert_Auswahl_Rechts">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>500</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BtDarstellen_Links">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>470</y>
      <width>93</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Darstellen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>470</y>
      <width>93</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Darstellen</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lbl_Mac_Vel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Höchstgeschwindigkeit :</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lbl_Weg">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>590</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Zurückgelete Distanz :</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lbl_Max_Acc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Größte Beschleunigung :</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lbl_Dauer">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>650</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dauer :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1457</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
